CONTRIBUTIONS.md

```diff
@@ -23,27 +23,27 @@ Moonbeam is following the
-* [prettier](https://prettier.io/) - run with `npx prettier --check --ignore-path .gitignore '**/*.(yml|js|ts|json)'` (runs against `typescript` code)
+- [clippy](https://github.com/rust-lang/rust-clippy) - run with `cargo clippy --release --workspace`
+- [rustfmt](https://github.com/rust-lang/rustfmt) - run with `cargo fmt -- --check`
+- [editorconfig](https://editorconfig.org/) - integrate into your text editor / IDE
+- [prettier](https://prettier.io/) - run with `npx prettier --check --ignore-path .gitignore '**/*.(yml|js|ts|json)'` (runs against `typescript` code)
 
 ### Directory Structure
 
 The following is a list of directories of interest in development.
 
-|Directory              |Purpose                                                                     |
-| --------------------- | -------------------------------------------------------------------------- |
-|client/                | Debug & Trace related code (rust)                                          |
-|docker/                | Dockerfiles for running Moonbeam                                           |
-|moonbeam-types-bundle/ | PolkadotJs types definitions for Moonbeam (typescript)                     |
-|node/                  | Moonbeam's main node (rust)                                                |
-|pallets/               | Moonmeam's Substrate runtime pallets (rust)                                |
-|primitives/            | More Debug & Trace related code (rust)                                     |
-|runtime/               | Moonbeam's runtime (on-chain) code (rust, compiled to WASM)                |
-|scripts/               | Utilities for launching and interacting with a Moonbeam chain (typescript) |
-|specs/                 | Spec files used to generate genesis for well-known Moonbeam networks       |
-|tools/                 | Various tools generally related to development (typescript)                |
+| Directory              | Purpose                                                                    |
+| ---------------------- | -------------------------------------------------------------------------- |
+| client/                | Debug & Trace related code (rust)                                          |
+| docker/                | Dockerfiles for running Moonbeam                                           |
+| moonbeam-types-bundle/ | PolkadotJs types definitions for Moonbeam (typescript)                     |
+| node/                  | Moonbeam's main node (rust)                                                |
+| pallets/               | Moonmeam's Substrate runtime pallets (rust)                                |
+| primitives/            | More Debug & Trace related code (rust)                                     |
+| runtime/               | Moonbeam's runtime (on-chain) code (rust, compiled to WASM)                |
+| scripts/               | Utilities for launching and interacting with a Moonbeam chain (typescript) |
```

Comment:
how do I revert these formatting changes 😭 ? I was trying to run prettier and it changed a lot of formatting

Reply:
thank you alan